### PR TITLE
Spark Content Style Guide

### PR DESCRIPTION
**Pull request by diriquelme**
## What does this PR do?
Capitalization and quotation guides

### Associated Issue
2501768 [UX] Add to Doc Site: Content Style Guidelines: Capitalization and quotation

## Please check off completed items as you work.
If a checklist item or section does not apply to your PR then please remove it.

### Documentation
 - [x] Update Spark Docs

### Deploy Preview Reviewed and Approved
 - [x] Design Team
 - [x] Accessibility Team

### Screenshots
Add screenshots to help explain your PR if you'd like. However, this is not
expected.

### CODE REVIEW

File: src/pages/using-spark/guides/content-style.mdx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  </li>
+</ul>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+## Capitalization
+
+Use for visual presentation of pieces of text to make it easy for users to read.
+
+> As a general rule, use **start case** for **headers**, use either **start case** or **sentence case** for **drop down options** and **labels**, depending on the context.
```
Comment (bobwise) [inline src/pages/using-spark/guides/content-style.mdx:52]:
```suggestion
> As a general rule, use **start case** for **headers**, use either **start case** or **sentence case** for **dropdown options** and **labels**, depending on the context.
```

**Review by bobwise — COMMENTED** [review]
Great job!

File: src/pages/using-spark/guides/content-style.mdx (hunk 1 above)
Comment (diriquelme) [reply src/pages/using-spark/guides/content-style.mdx]: Thank you @bobwise :)

File: src/pages/using-spark/guides/content-style.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+- Homeowners Insurance Info
+- How To Buy A Home
+- Lock Your Rate And Save
+- Check Your To Do List
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+### Sentence Case
+
+Sentence case capitalizes the first word of the sentence and proper nouns.
```
Comment (RVMendoza) [inline src/pages/using-spark/guides/content-style.mdx:78]: Sentence case capitalizes the first word of each sentence as well as proper nouns.

**Review by bobwise — APPROVED** [review]

**Review by RVMendoza — APPROVED** [review]

**Review by guern1kn — APPROVED** [review]

**Comment by RVMendoza** [comment]
tenor-199570195

**Pull request merged**